content/components/action-list.mdx

```diff
@@ -248,3 +249,9 @@ A nav list organizes navigation links for the user's current context and indicat
 </div>
 
 </Box>
+
+## Accessibility
+
+### Known accessibility issues (GitHub staff only)
+
+ <AccessibilityLink label="ActionList"/>
```

Comment:
The heading followed by the link feels a little repetitive, but not worth blocking the PR over (especially if we're automating this soon)